app/views/users/index.html.erb

```diff
@@ -1,16 +1,15 @@
-
-<ul id='users_section'>
+<div id='users_section'>
```

Comment:
- [ ] Your application is not running as intended. When your application is launched it displays a blank screen, with no content. Kindly ensure to improve upon this aspect